dashboard/app/models/user_level.rb

```diff
@@ -24,6 +26,10 @@ class UserLevel < ActiveRecord::Base
   belongs_to :level
   belongs_to :script
 
+  scope :attempted, -> { where.not(best_result: nil) }
+  scope :passed, -> { where('best_result >= ?', ActivityConstants::MINIMUM_PASS_RESULT) }
```

Comment:
would it make sense to make this passing and best for naming consistency? (or change passing and best if you like these names better).